internal/services/healthcare/sdk/2021-01-11/collection/client.go

```diff
@@ -0,0 +1,15 @@
+package collection
+
+import "github.com/Azure/go-autorest/autorest"
+
+type CollectionClient struct {
+	Client  autorest.Client
+	baseUri string
+}
+
+func NewCollectionClientWithBaseURI(endpoint string) CollectionClient {
+	return CollectionClient{
+		Client:  autorest.NewClientWithUserAgent(userAgent()),
+		baseUri: endpoint,
+	}
+}
```

Comment:
Does the pandora sdk need to be embedded here? I don't see it being used anywhere.

Reply:
Thanks @stephybun , removed the SDK